lib/modules/manager/bazel-module/extract.spec.ts

```diff
@@ -1,72 +1,60 @@
 import { codeBlock } from 'common-tags';
+import upath from 'upath';
+import { Fixtures } from '../../../../test/fixtures';
+import { GlobalConfig } from '../../../config/global';
+import type { RepoGlobalConfig } from '../../../config/types';
 import { BazelDatasource } from '../../datasource/bazel';
 import { GithubTagsDatasource } from '../../datasource/github-tags';
 import * as parser from './parser';
 import { extractPackageFile } from '.';
 
+const adminConfig: RepoGlobalConfig = {
+  localDir: upath.resolve('lib/modules/manager/bazel-module/__fixtures__'),
+};
+
```

Comment:
i would prefer to mock util/fs, so you can also assert that specific files are read.

Reply:
I switched over the fixtures in the `bazelrc.spec.ts` to use mocks, since that is where I am exercising all of the import permutations. I left a single fixture/test in `extract.spec.ts` as an integration test.